src/components/EntityInfo/EntityHeader.vue

```diff
@@ -8,10 +8,8 @@
 
       <!-- Subtitle -->
       <div>
-        <span id="business-description">{{ businessDescription }} </span>
-        <span id="active-util-wrap" class="ml-3"  v-if="isInLimitedRestoration">
-          <span id="active-util" class="ml-3">Active until {{ getLimitedRestorationActiveUntil || 'Unknown' }}</span>
-        </span>
+        <span id="business-description">{{ businessDescription }}</span>
+        <span id="active-util" class="ml-3 pl-3" v-if="isInLimitedRestoration">Active until {{ getLimitedRestorationActiveUntil || 'Unknown' }}</span>
```

Comment:
This line is too long.

Make sure you run `npm run lint` locally before pushing commits.

In the near future, I think we are going with 1 property per line for most HTML elements. I think that would work well here now.

Reply:
Fixed, thanks.